src/modelgauge/tests/safe_v1.py

```diff
@@ -131,15 +138,25 @@ def _check_annotators(cls):
         if not hasattr(cls, "annotators") or cls.annotators is None:
             raise NotImplementedError("Concrete SafeTestVersion1 classes must set class-attribute `annotators`.")
 
+    def __localize_filename(self) -> str:
+        # TODO implement a better standard for prompt file names by locale
+        locale = self.locale.value.lower()
+        if locale and locale != "en_us":
+            tail = f"_{locale}"
+        else:
+            tail = ""
+        filename = f"{self.prompt_set_file_name}{tail}"
+        return filename
+
     def get_dependencies(self) -> Mapping[str, ExternalData]:
-        modellab_base_download_url = "https://modellab.modelmodel.org/files/download"
+        prompt_set_base_download_url = f"https://{PROMPT_SET_DOWNLOAD_HOST}/files/download"
         headers = None
         if self.token is not None:
             headers = {"auth-token": self.token.value}
-        # Only one dependency.
+        source_url = f"{prompt_set_base_download_url}/{self.__localize_filename()}.csv"
         return {
             self.prompt_set_file_name: WebData(
```

Comment:
It currently doesn't include the locale, because that string is also the "family" of prompts (the set of prompts from a specific vendor at a specific version, with a practice vs. official flag). The prompt file can contain multiple locales. The file name string is a value in a dict key with two keys ("official" and "practice") that are used in other apps as well (modellab and modelrunner). Having multiple files (e.g. the same vendor/version of prompts but different locales) with distinct names would require changing modellab and modelrunner too, and the way dependencies are managed. I tried that approach and it got ugly quickly. There's more detail in the PR description.